mmv1/templates/tgc/resource_converters.go.erb

```diff
@@ -92,7 +92,9 @@ func ResourceConverters() map[string][]cai.ResourceConverter {
 		"google_storage_bucket_iam_policy":                {resourceConverterStorageBucketIamPolicy()},
 		"google_storage_bucket_iam_binding":               {resourceConverterStorageBucketIamBinding()},
 		"google_storage_bucket_iam_member":                {resourceConverterStorageBucketIamMember()},
-    "google_cloud_tasks_queue":                        {cloudtasks.ResourceConverterCloudTasksQueue()},
+		"google_compute_node_group":                       {compute.ResourceConverterComputeNodeGroup()},
+		"google_compute_node_template":                    {compute.ResourceConverterComputeNodeTemplate()},
```

Comment:
This is still adding node_template. Please remove this line.

Reply:
if I remove google_compute_node_template, it will not test node template resource anymore. I can't remove it.

Reply:
google_compute_node_template support was added in https://github.com/GoogleCloudPlatform/magic-modules/pull/10602, so it doesn't also need to be added here.

Reply:
ok, done. Thanks